client/containers/DashboardOverview/overviewRows/labels.tsx

```diff
@@ -78,3 +84,101 @@ export const getPubReleasedStateLabel = (pub: Pub) => {
 		icon: 'lock' as const,
 	};
 };
+
+export const getSubmissionStatusLabel = (
+	submissionStatus: Partial<SubmissionStatus>,
+): IconLabelPair => {
+	if (submissionStatus === 'accepted') {
+		return {
+			label: 'Accepted',
+			icon: 'symbol-square',
+			intent: 'success',
+		};
+	}
+	if (submissionStatus === 'declined') {
+		return {
+			label: 'Declined',
+			icon: 'symbol-square',
+			intent: 'danger',
+		};
+	}
+	if (submissionStatus === 'pending') {
+		return {
+			label: 'Pending',
+			icon: 'symbol-square',
+			intent: 'warning',
+		};
+	}
+	return {
+		label: 'Incomplete',
+		icon: 'symbol-square',
+		intent: 'none',
+	};
+};
+
+export const getSubmissionTimeLabel = (pub: Pub) => {
+	const submissiondDate = getPubSubmissionDate(pub);
+	if (submissiondDate) {
+		return {
+			label: (
+				<>
+					<em>Submitted</em>&nbsp;{getDateLabelPart(submissiondDate)}
+				</>
+			),
+			icon: 'time' as const,
+		};
+	}
+	return {
+		label: 'Update Status',
+		icon: 'warning-sign' as const,
+	};
+};
+
+export const renderLabelPairs = (iconLabelPairs: IconLabelPair[]) => {
+	return (
+		<div className="summary-icons">
+			{iconLabelPairs.map((iconLabelPair, index) => {
+				const {
+					icon,
+					label,
+					iconSize: iconLabelPairIconSize = 12,
+					intent = 'none',
+				} = iconLabelPair;
+				const iconElement =
+					typeof icon === 'string' ? (
+						<Icon icon={icon} iconSize={iconLabelPairIconSize} intent={intent} />
+					) : (
+						icon
+					);
+				return (
+					<div
+						className="summary-icon-pair"
+						// eslint-disable-next-line react/no-array-index-key
+						key={index}
+					>
+						{iconElement}
+						{label}
+					</div>
+				);
+			})}
+		</div>
+	);
+};
+
+export const renderDetailsRow = (pub: Pub, hasSubmission: boolean): React.ReactNode => {
```

Comment:
Okay, I see the problem now with my suggestion to call this `hasSubmission` — the obvious check to see where a pub "has submission" is just to check `pub.submission`. I guess I'd call this something like `showSubmissionDetails`?

Reply:
pretty much, i just didn't want to have something look like this

```
...
if (pub.submission) {
     const { status } = pub.submission;
     const detailsRow = pub.submission
	     ? renderLabelPairs([getSubmissionStatusLabel(status), getSubmissionTimeLabel(pub)])
	     : renderLabelPairs([
	           ...getScopeSummaryLabels(pub.scopeSummary),
	           getPubReleasedStateLabel(pub),
	     ]);
     return detailsRow;
......
```
because a pub may have a submission, but render in the collections or community overview tab. explicitly passing isSubmission seems to sign post if the `PubOverviewRow ` component being rendered is from the `SubmissionOverview` render the submissions details row. hasSubmission feels more natural. 

`renderDetailsRow` is rendering and `IconLabelPair` (or whatever else we want there) wherever `PubOverviewRow` is being called so what about `renderDetails`?